pages.pt_BR/common/curl.md

```diff
@@ -1,37 +1,37 @@
 # curl
 
 > Transfere dados entre o computador local e um servidor remoto.
```

Comment:
Removing "remoto" as there is no "remote" in the original file and curl can act on local servers as well, so it doesn't seem to be appropriate to add "remoto" in the translation. Thanks @renie for spotting this.

```suggestion
> Transfere dados entre o computador local e um servidor.
```